physics/smoke_dust/dep_dry_mod_emerson.F90

```diff
@@ -220,25 +234,25 @@ subroutine dry_dep_driver_emerson(rmol,ustar,znt,ndvel,ddvel,vgrav,   &
                IF (ndt_settl(nv) > 12) ndt_settl(nv) = 12 
                dt_settl(nv) = REAL(ntdt,kind=kind_phys) /REAL(ndt_settl(nv),kind=kind_phys)
              enddo
-             do nv = 1, ndvel
-               chem_before(nv) = 0._kind_phys
-               do k = kts, kte
-                chem_before(nv) = chem_before(nv) + (cblk_col(k,nv) * rho_phy(i,k,j) * delz(i,k,j) ) ! ug/m2
-              enddo
-             enddo
+             !do nv = 1, ndvel
```

Comment:
Consider removing the commented lines entirely since one can always retrieve code from previous commits in the git history.

Reply:
@grantfirl I will try my best to remove the commented lines. Thanks.